### PR TITLE
Adds contents to tower-caps and steel-caps

### PR DESCRIPTION
**Pull request by MrHorizons**
Due to their previous lack of such, tower-caps and steel-caps now have reagent contents, of, respectively, 50% carbon and 50% iron.

#### Changelog

:cl:  
rscadd: Tower-caps contain 50% carbon and steel-caps contain 50% iron.
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
lol

**Comment by MrHorizons** [comment]
> lol

lol

File: yogstation.dme [hunk 1]
```diff
@@ -988,6 +988,7 @@
 #include "code\game\objects\items\implants\implant_greytide.dm"
 #include "code\game\objects\items\implants\implant_krav_maga.dm"
 #include "code\game\objects\items\implants\implant_mindshield.dm"
+#include "code\game\objects\items\implants\implant_mindshieldtot.dm"
```
Comment (JamieD1) [inline yogstation.dme]:
```suggestion
```

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: yogstation.dme [hunk 2]
```diff
@@ -998,7 +998,6 @@
 #include "code\game\objects\items\implants\implanter.dm"
 #include "code\game\objects\items\implants\implantpad.dm"
 #include "code\game\objects\items\implants\implantuplink.dm"
```
Comment (JamieD1) [inline yogstation.dme]:
```suggestion
#include "code\game\objects\items\implants\implantuplink.dm"
#include "code\game\objects\items\implants\implant_mindshieldtot.dm"
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**